draft-terbu-sd-jwt-vc.md

```diff
@@ -83,27 +83,31 @@ To support revocation of Verifiable Credentials, an optional fourth party can be
 involved, a Status Provider, who delivers revocation information to Verifiers.
 (The Verifier can also serve as the Status Provider.)
 
-This specification defines Verifiable Credentials based on the SD-JWT
-format with a JWT Claim Set.
+This specification defines Verifiable Credentials based on JWT and SD-JWT.
 
 ## Rationale
 
-JSON Web Tokens (JWTs) [@!RFC7519] can in principle be used to express
-Verifiable Credentials in a way that is easy to understand and process as it
-builds upon established web primitives. While JWT-based credentials enable selective
+JSON Web Tokens (JWTs) [@!RFC7519] can be used to express
+Verifiable Credentials in the three-party-model
+in a way that is easy to understand and process as it
+builds upon established web primitives.
+
+JWT-based credentials enable selective
```

Comment:
I suggest to remove this paragraph. It is not required in this draft.

Reply:
Agreed I think we can remove this.

Reply:
removed